pages/statistics.jsx

```diff
@@ -0,0 +1,28 @@
+import Header from "@/components/Header";
+import Stat from "@/components/Stat";
+import styles from "@/styles/pages/statistics.module.scss";
+
+export default function Statistics() {
+	const data = [
+		{ amount: 19, name: "all asks" },
+		{ amount: 7, name: "my asks" },
+		{ amount: 3, name: "liked asks" },
+		{ amount: 3, name: "accounts" },
+		{ amount: 1, name: "accounts" },
+		{ amount: 1, name: "accounts" },
+		{ amount: 1, name: "accounts" },
+	];
+
+	return (
+		<>
+			<Header />
```

Comment:
This appears in every page. You can render it in `app.jsx` so it's part of the site's layout

Reply:
noted